modules/deploy/pages/deployment-option/cloud/api/cloud-controlplane-api.adoc

```diff
@@ -0,0 +1,184 @@
+= Manage Clusters using the Cloud API
+:description: Use the Cloud API to manage your clusters and organization-level resources in Redpanda Cloud.
+:page-cloud: true
+
+The Redpanda Cloud API is a collection of REST APIs that allow you to interact with different parts of Redpanda Cloud. The Control Plane API enables you to programmatically manage your clusters outside of the Redpanda Cloud UI. You can call the API endpoints directly, or use tools like Terraform or Python scripts to automate cluster management.
+
+== Control Plane API
+
+The Control Plane API is one central API that allows you to provision clusters and networks.
```

Comment:
The Control Plane API allows you to provision clusters, networks and namespaces.